jenkins/PerfCI_Chaos/04_PerfCI_Chaos_Upgrade_OpenShift_Deployment/Jenkinsfile

```diff
@@ -132,156 +129,409 @@ END
-                                        -e GOOGLE_DRIVE_TOKEN_FILE='${GOOGLE_DESTINATION_PATH}/${GOOGLE_TOKEN_FILE}' \
-                                        -e GOOGLE_DRIVE_SHARED_DRIVE_ID='${GOOGLE_DRIVE_SHARED_DRIVE_ID}' \
-                                        -e RUN_ARTIFACTS_URL='${RUN_ARTIFACTS_URL}' \
-                                        -e BUILD_VERSION='${build_version}' \
-                                        -e RUN_TYPE='${RUN_TYPE}' \
-                                        -e SAVE_ARTIFACTS_LOCAL=${SAVE_ARTIFACTS_LOCAL} \
-                                        -e ENABLE_PROMETHEUS_SNAPSHOT=${ENABLE_PROMETHEUS_SNAPSHOT} \
-                                        -e WORKER_DISK_IDS='${WORKER_DISK_IDS}' \
-                                        -e WORKER_DISK_PREFIX='${WORKER_DISK_PREFIX}' \
-                                        -e SCALE='${SCALE}' \
-                                        -e SCALE_NODES='${SCALE_NODES}' \
-                                        -e REDIS='${REDIS}' \
-                                        -e THREADS_LIMIT='${THREADS_LIMIT}' \
-                                        -e WINDOWS_URL='${WINDOWS_URL}'  \
-                                        -e DELETE_ALL=${DELETE_ALL} \
-                                        -e VERIFICATION_ONLY=${VERIFICATION_ONLY} \
-                                        -e LSO_NODE='${LSO_NODE}'  \
-                                        -e TIMEOUT='${TIMEOUT}' \
-                                        -e log_level='INFO' \
-                                        -v '${KUBECONFIG_PATH}:${KUBECONFIG_PATH}' \
-                                        -v '${TEMP_DIR}:${GOOGLE_DESTINATION_PATH}' \
-                                        --privileged '${QUAY_BENCHMARK_RUNNER_REPOSITORY}'
+                                        sudo podman run --rm \\
+                                            -e UPGRADE_OCP_VERSION="${UPGRADE_OCP_VERSION}" \\
+                                            -e UPGRADE_STEP='verify_bare_metal_upgrade_complete' \\
+                                            -e RUN_TYPE="${RUN_TYPE}" \\
+                                            -e UPGRADE_CHANNEL="${UPGRADE_CHANNEL}" \\
+                                            -e LSO_VERSION="${LSO_VERSION}" \\
+                                            -e ODF_VERSION="${ODF_VERSION}" \\
+                                            -e CNV_VERSION="${CNV_VERSION}" \\
+                                            -e EXPECTED_NODES='${EXPECTED_NODES}' \\
+                                            -e KUBEADMIN_PASSWORD="${KUBEADMIN_PASSWORD}" \\
+                                            -e TIMEOUT="${TIMEOUT}" \\
+                                            -v "${KUBECONFIG_PATH}":"${KUBECONFIG_PATH}" \\
+                                            --privileged "${QUAY_BENCHMARK_RUNNER_REPOSITORY}"
```

Comment:
Why the doubled backslash (this isn't done anywhere else)?

Reply:
Good catch: fixed